src/vue/components/nav/drop-down-menu.vue

```diff
@@ -14,7 +14,7 @@
     </div>
 
     <div class="alpheios-navmenu__item" :class="{ active: $store.getters['ui/isActiveTab']('definitions') }"
-         @click="changeTab('definitions')" v-show="$store.getters['app/defDataReady']">
+         @click="changeTab('definitions')" v-show="$store.getters['app/fullDefDataReady']">
```

Comment:
I think, that we publish here short defs too, may be both checks? (all the following comments are the same - I am checking from bottom to the top - because remembered it only on the popup file)

Reply:
see previous comment